<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CM_MainWindow</class>
 <widget class="QMainWindow" name="CM_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doubly Linked List GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CM_GraphicsView" name="widget_gv" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFind_First_Of"/>
    <addaction name="actionFind_Last_Of"/>
    <addaction name="actionConstruct_Random_String"/>
    <addaction name="actionClear_List"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_use_this_GUI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFind_First_Of">
   <property name="text">
    <string>Find First Of</string>
   </property>
  </action>
  <action name="actionClear_List">
   <property name="text">
    <string>Clear List</string>
   </property>
  </action>
  <action name="actionFind_Last_Of">
   <property name="text">
    <string>Find Last Of</string>
   </property>
  </action>
  <action name="actionConstruct_Random_String">
   <property name="text">
    <string>Construct Random String</string>
   </property>
  </action>
  <action name="actionHow_to_use_this_GUI">
   <property name="text">
    <string>How to use this GUI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CM_GraphicsView</class>
   <extends>QWidget</extends>
   <header>CM_GraphicsView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
